### ISSUE TITLE
Requests fail

### ISSUE THREAD

**Issue by maelchiotti** [fedecalendino/wrap-genius#1]
I just noticed that any request seems to fail. I tried the example in the readme, and I always get the same exception: `requests.exceptions.JSONDecodeError: [Errno Expecting value]`. Full traceback here : [traceback.txt](https://github.com/fedecalendino/wrap-genius/files/9439899/traceback.txt).

**Comment by fedecalendino** [comment]
Hi @maelchiotti!

I had no exceptions running the example on my laptop, tho I saw that the lyrics scraper is not working as intended.

Would you be so kind to send me exactly what you are running?

Also, were you able to create an ACCESS TOKEN for Genius' api?


Fede

**Comment by maelchiotti** [comment]
Thanks for the quick reply!

The exact code I am running is:
```python
from genius import Genius
g = Genius(access_token="token")
artist = g.search_artist("Gorillaz")
print(artist)
```

I was able to ceate an access token. I have used you project for many months. I was on vacation for a few weeks, and I just tried it again now. I guess I'm doing something wrong...

**Comment by maelchiotti** [comment]
Ok, so I tried what I should have tried earlier: removing and adding back your package. Now it works perfectly... Thanks again for your great work, I use it in one of my first projects and it works great!

**Comment by maelchiotti** [comment]
I too have noticed that the lyrics scrapper doesn't work, that's why I used [LyricsGenius](https://github.com/johnwmillr/LyricsGenius) in my project. But it would be awesome if it did!

**Comment by fedecalendino** [comment]
@maelchiotti lyrics parser should be fixed, in **v1.7.1**.

```python
song = g.get_song(song_id=3027414)
print("Song:", song.title_with_featured)
print(" * released date:", song.release_date_for_display)
print(" * album:", song.album)

print()
print("Artist:", song.artist)
print("Features:")
for featured in song.features:
    print("   -", featured.name)

lyrics = song.lyrics
print()
print("Lyrics:")
print('\n'.join(lyrics))
```

```text
Song: Andromeda (Ft. Shelley FKA DRAM)
 * released date: March 23, 2017
 * album: Humanz (Deluxe) (661004)

Artist: Gorillaz (860)
Features:
   - Shelley FKA DRAM

Lyrics:
[Verse 1: 2-D]
When the pulsing looks to die for
Take it in your heart now, lover
When the case is out
And tired and sodden
Take it in your heart
Take it in your heart
[Pre-Chorus: Shelley FKA DRAM]
Back to when it was cool
'Cause there's no substitute
Who even knows the truth?
The truth, the truth
[Chorus: 2-D]
Take it in your heart now, love her
Take it in your heart, heart, heart, heart
Take it in your heart
Take it in your heart, heart, heart, heart
Where it all goes down
[Verse 2: 2-D]
Outside, cold and ghosting out with jet lag
I took it to the right man
Took it all back
When the courts were closing
It was Bobby gracing
I know that
I put it to the right man
He put it back
[Bridge: 2-D]
Talk to the crowds
Stacks of lights
Come streaming back
Make it for the best times
Growing pains, good times
Good times, good times
Good times, good times
Good times, good times
```

**Comment by maelchiotti** [comment]
That's awesome!
